config/holes.toml

```diff
@@ -1981,6 +1981,49 @@ Then I'm sure she sells seashore shells.
 </blockquote>
 '''
 
+[Turtle]
+category = "Art"
+experiment = -1
```

Comment:
You actually have an issue, may as well use it: `experiment: 622`

Reply:
oh that's what it is

Reply:
ok i added it

Reply:
Yep, the -1 is actually quite hacky, the template does this:
```
<p>
    This hole is experimental, no solutions will be saved.
{{ if ne . -1 }}
    Please leave feedback on the
    <a href="//github.com/code-golf/code-golf/issues/{{ . }}">
        GitHub issue</a>.
{{ end }}
```